jupyterlite_xeus/create_conda_env.py

```diff
@@ -115,8 +118,10 @@ def _create_conda_env_from_specs_impl(env_name, root_prefix, specs, channels):
                 *specs,
             ],
             check=True,
+            capture_output=True,
         )
-        return
+        for entry in [entry for entry in json.loads(res.stdout)["actions"]["LINK"] if entry["name"] == "python"]:
```

Comment:
The only approach could be to inspect the created environment to see which Python version was installed. The current state of your PR only works if micromamba was used.

Reply:
Did you see it is in draft?

Reply:
Sure! I'm only suggesting inspecting the created environment instead of inspecting the logs. 

I am not sure there is a real clean solution here, although inspecting the logs may be a bit fragile.

Reply:
Thanks for the feedback.
Well it is what I do here, it inspects the environment that was just created. `micromamba` at least offers some (JSON) structured data in `stdout`, that is easy and safe to parse. I opened some issues in the past (https://github.com/mamba-org/mamba/issues/1922 and https://github.com/mamba-org/mamba/issues/1919), so that we could use the Python API to get such information. I need to check if this was ever implemented.

Reply:
Unfortunately we cannot use `libmambapy` just yet (see https://github.com/mamba-org/mamba/issues/1922#issuecomment-2621694611). But from what I can see, `conda install` also support `--json`, so I guess we can just use that for all "conda-like` managers?